examples/monitor.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2018 The rust-gpio-cdev Project Developers.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate gpio_cdev;
+extern crate nix;
+#[macro_use] extern crate quicli;
+
+use gpio_cdev::*;
+use quicli::prelude::*;
+use nix::poll::*;
+use std::os::unix::io::{AsRawFd};
+
+type PollEventFlags = nix::poll::EventFlags;
+
+#[derive(Debug, StructOpt)]
+struct Cli {
+    /// The gpiochip device (e.g. /dev/gpiochip0)
+    chip: String,
+    /// The offset of the GPIO lines for the provided chip
+    lines: Vec<u32>,
+}
+
+fn do_main(args: Cli) -> errors::Result<()> {
+    let mut chip = Chip::new(args.chip)?;
+
+    // Get event handles for each line to monitor.
+    let mut evt_handles: Vec<LineEventHandle> = args.lines.into_iter().map(|off| {
+        let line = chip.get_line(off).unwrap();
```

Comment:
nit: Since we return result here you can just use `?` instead of including the `unwrap`.

Reply:
Ah, nevermind, missed the closure on first glance.